application/lib/model/Task.dart

```diff
@@ -6,10 +10,38 @@ enum Status {
 
 enum Priority { low, medium, high, critical }
 
-class Task {
-  Task({this.taskName, this.taskStatus, this.taskPriority, this.newInformation = false});
+class Task implements Json {
+  Task({this.taskName, this.appId, this.screenId, this.taskDescription = ''});
+  @override
+  Task.fromJson(Map<String, dynamic> json)
+      : taskName = json['taskName'],
+        appId = json['appId'],
+        taskDescription = json['description'],
+        screenId = json['screenId'];
```

Comment:
Får denne fejl: _TypeError (type 'List<dynamic>' is not a subtype of type 'List<int>'), når jeg logger ind på appen.